draft-ietf-masque-connect-ip.md

```diff
@@ -1331,6 +1331,36 @@ and outer IP headers does not apply because the outer connection will react
 correctly to congestion notifications if it uses ECN. The inner traffic can
 also use ECN, independently of whether it is in use on the outer connection.
 
+## Diffserv Considerations {#dscp-consideration}
+
+Tunneled IP packets can have Diffserv Code Points (DSCP) {{!DSCP=RFC2474}} set
+in the traffic class IP header field to request a particular per-hop
+behavior. If a client or IP proxy handling an IP proxying request is configured
+as part of a diffserv domain, it MAY implement traffic differentiation. However,
+the use of HTTP can limit the possibilities for differentiated treatment of the
+tunneled IP packets on the path between the ingress and egress.
+
+If a client or IP proxy with a connection containing an IP Proxying request
+stream is using congestion control for that stream, regardless of whether HTTP
+datagrams or capsules are used, all traffic without of DSCP markings will be
+treated equally within that transport connection. The client or proxy MUST NOT
```

Comment:
```suggestion
If tunneled packets are subject to congestion control, they will be treated
equally regardless of their DSCP markings. IP proxying endpoints MUST NOT
```

Reply:
I realize that my sentence can be shorten but it actually lacks an important distinction. It states the wrong chain of dependency. Using congestion control requires one to not expose different DSCP within the connection avoid screwing up the congestion controller. So maybe formulating this as: 

If tunneled packets are subject to congestion control, they will need to be treated
equally, i.e. use the same DSCP in the tunnelling connection, regardless of the encapsulated DSCP markings.

Is that better?

Reply:
I committed another rewording that I hope improves things without loosing relevant details.